stakingModel/contracts/UBIScheme.sol

```diff
@@ -163,6 +184,24 @@ contract UBIScheme is AbstractUBI {
         return false;
     }
 
+    /* @dev Transfers `amount` dao tokens to `account`. updates stats
+     * and emits an events.
+     * @param account the account which recieves the funds
+     * @param amount the amount to transfer
+     */
+    function _transferClaimedTokens(address account, uint256 amount)
+        private
+        requireActive
+    {
+        Day memory day = claimDay[currentDay];
+        day.amountOfClaimers = day.amountOfClaimers.add(1);
+        day.claimAmount = day.claimAmount.add(amount);
+        claimDay[currentDay] = day;
```

Comment:
why not use Day storage so you dont have to re-assign

Reply:
updated 420b81e35ef9989dca39047c7a76b2844ff4e825